addons/hr_holidays/wizard/hr_leave_generate_multi_wizard.py

```diff
@@ -3,14 +3,20 @@
 from datetime import datetime, timedelta
 from pytz import timezone, UTC
 
-from odoo import fields, models, _
-from odoo.exceptions import UserError
+from odoo import fields, models, api
+from odoo.exceptions import UserError, AccessError
```

Comment:
```suggestion
from odoo import api, fields, models
from odoo.exceptions import AccessError, UserError
```
for alphabetical order ;)